apps/kube-prometheus-stack/defaults.yaml

```diff
@@ -0,0 +1,9 @@
+prometheus:
```

Comment:
hmm, I think this is annoyingly in the wrong repo (the base cluster repo installs the kube-promtheus-stack, I think). I think there is a CRD you can add to get this scrape to happen instead?